### DISCUSSION TITLE
How can I change the default Marker Icon

### DISCUSSION THREAD

**Discussion by CityBelfast** [geoman-io/leaflet-geoman#855] [Questions & Help]
Hi, I'm unable to change the icon of place marker eve though I could change the icon of default marker. is there any way where we can change place marker as tool  marker icon?

![leaflet-marker-issue](https://user-images.githubusercontent.com/76213912/102620558-1453ae80-4136-11eb-9ffc-52d350159155.png)

**Comment by Falke-Design** [comment]
Use the option `markerStyle: {}`

```
var icon = L.icon({
	iconUrl: 'https://i.pinimg.com/originals/25/62/aa/2562aacd1a4c2af60cce9629b1e05cf2.png',
	iconSize: [40,40],
	iconAnchor: [20,40]
})

map.pm.setGlobalOptions({markerStyle: {icon: icon}})
// or
// map.pm.enableDraw('Marker',{markerStyle: {icon: icon}})
```

**Comment by CityBelfast** [comment]
Thanks a lot for the quick reply
